### DISCUSSION TITLE
Using whole touchpad area for scrolling when Air mouse is active.

### DISCUSSION THREAD

**Discussion by Polisson** [AppGround-io/bluetooth-keyboard-and-mouse-support#117] [Ideas]
Air mouse functionality is awesome. I've met such features years ago on a little remote-control Android app (then it spreaded to many others).
... but using air mouse with touch mouse simultaneous is confusing.
Ideally, I imagine: 
when air mouse is enabled, the whole touchpad area is for 
- single finger scrolling (vertically and horizontally), 
- tapping/clicking (single, short for LMB, double or long for RMB, tripple for MMB),
- making other gestures (eg. triple swipe).

**Comment by gotjoshua** [comment]
I don't have premium yet, but trying to understand what is airmouse?
why is it awesome?
where can i read more about it?

**Reply by Staini** [reply]
it's like a Wiimote without IR...
